cmd/download_test.go

```diff
@@ -114,7 +114,15 @@ func TestDownload(t *testing.T) {
 		err = runDownload(cfg, flags, []string{})
 		assert.NoError(t, err)
 
-		assertDownloadedCorrectFiles(t, filepath.Join(tmpDir, tc.expectedDir), tc.requestor)
+		targetDir := filepath.Join(tmpDir, tc.expectedDir)
+		assertDownloadedCorrectFiles(t, targetDir, tc.requestor)
+
+		metadata := `{"track":"bogus-track","exercise":"bogus-exercise","id":"bogus-id","url":"","handle":"alice","is_requester":%s,"auto_approve":false}`
+		metadata = fmt.Sprintf(metadata, tc.requestor)
+		path := filepath.Join(targetDir, "bogus-track", "bogus-exercise", ".solution.json")
```

Comment:
For some reason I was under the impression that downloading the same exercise twice would result in different directories. Specifically `exercise` and `exercise-2`. But the CLI doesn't add a suffix, it just supports reading from a suffix.

Reply:
If they have different solution UUIDs, then they will download to separate directories. I think we have a bug in the separate directory thing, so I might strip that out until we need to support that feature.